### PR TITLE
Improvement/gero 337 gene info related genes speedup

### PR DESCRIPTION
**Pull request by dustinbleile**
https://www.bcgsc.ca/jira/browse/GERO-337

Some speedup and improve description of get_gene_information

### CODE REVIEW

File: ipr/annotate.py [hunk 1]
```diff
@@ -34,69 +33,78 @@
...
+        'conditions.@rid',
+        'conditions.@class',
+        'conditions.reference1',
+        'conditions.reference2',
+        'reviewStatus',
+    ]
+    body: Dict[str, Any] = {'target': 'Statement', 'returnProperties': ret_props}
+
+    gene_names = sorted(set(gene_names))
+    statements = graphkb_conn.query(body)
```
[the remark this answers is not in the document]
Comment (dustinbleile) [reply ipr/annotate.py:66]: Partly to be able to check and ignore the failed review status.

I'd also thought it seemed much faster. 
But I'm less sure now how real that effect was, given how much other variability query time I was seeing.

**Comment by dustinbleile** [comment]
Hi @elewis2, @mathieulemieux, @mattdoug604

We might want to move the get_gene_information function to graphkb/genes.py instead or next.
The other functions in pori_graphkb_python seem more directly related in most ways, though the final format of the list is specifically for IPR.

https://github.com/bcgsc/pori_graphkb_python/blob/master/graphkb/genes.py

**Comment by mathieulemieux** [comment]
> Hi @elewis2, @mathieulemieux, @mattdoug604
> 
> We might want to move the get_gene_information function to graphkb/genes.py instead or next. The other functions in pori_graphkb_python seem more directly related in most ways, though the final format of the list is specifically for IPR.
> 
> https://github.com/bcgsc/pori_graphkb_python/blob/master/graphkb/genes.py

Agree. But you already did work on it in IPR so maybe after merging these changes we go forward with mooving the function? What do you think?

**Comment by dustinbleile** [comment]
>..you already did work on it in IPR so maybe after merging these changes we go forward with moving the function?
@mathieulemieux 
Yeah, that is the idea and I wanted some comments on the return format anyway.

**Pull request closed without merging**